react/components/Table/index.js

```diff
@@ -139,6 +139,11 @@ class Table extends PureComponent {
     )
   }
 
+  handleInputSearchClearWithBulkAction = event => {
+    this.handleDeselectAllLines()
+    this.props.toolbar.inputSearch.onClear(event)
```

Comment:
No need to check if the function exists?

Reply:
We are already checking if the inputSearch exists in line 262 (265 to be more specific). The handleInputSearchClearWithBulkActions function in only being used when there are bulk actions and the input search.

Do you think it’s necessary to check if the onCleat function is defined there too?

Reply:
If there's any chance of the function `onClear()` to be called without being defined then yes. Otherwise no.